boilerplate/fullstack/next-multitenancy/config/frontend/multitenancy.tsx

```diff
@@ -4,6 +4,7 @@ import Multitenancy from "supertokens-auth-react/recipe/multitenancy";
 import { ThirdPartyEmailPasswordPreBuiltUI } from "supertokens-auth-react/recipe/thirdpartyemailpassword/prebuiltui";
 import { ThirdPartyPasswordlessPreBuiltUI } from "supertokens-auth-react/recipe/thirdpartypasswordless/prebuiltui";
 import SessionReact from "supertokens-auth-react/recipe/session";
+import { SuperTokensConfig } from "supertokens-auth-react/lib/build/types";
```

Comment:
I added the `SuperTokensConfig` type from the build folder to be consistent with other recipes. If we decide to fix this, we will do in it all the recipes at the same time.